### PR TITLE
chore: move auth-helper docs to guides.

### PR DESCRIPTION
**Pull request by thorwebdev**
- [x] Quick win: make auth helper guides available at `/docs/guides/auth/auth-helpers/`
- [ ] Fix typedoc generation in `auth-helpers`
- [ ] Replace https://supabase.com/docs/reference/auth-helpers with the typedoc generated reference
- [ ] Add redirects for the Next.js and SvelteKit guides

image

### CODE REVIEW

File: apps/reference/docs/guides/auth/auth-helpers/index.mdx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+id: index
+title: Overview
+description: A collection of framework specific Auth utilities for working with Supabase.
+---
+
+# Auth Helpers
+
+A collection of framework specific Auth utilities for working with Supabase.
```
Comment (dannykng) [inline apps/reference/docs/guides/auth/auth-helpers/index.mdx:9]: Makes this a bit more of a landing page:
Screen Shot 2022-09-19 at 1 39 47 PM

```suggestion
title: Auth Helpers
description: A collection of framework-specific Auth utilities for working with Supabase.
sidebar_label: Overview
---

import useBaseUrl from '@docusaurus/useBaseUrl'
import ButtonCard from '@site/src/components/ButtonCard'

A collection of framework-specific Auth utilities for working with Supabase.

<div class="container" style={{ padding: 0 }}>
  <div class="row is-multiline">
    {/* Auth UI */}
    <div class="col col--4">
      <ButtonCard
        class="card"
        to={useBaseUrl('/guides/auth/auth-helpers/auth-ui')}
        title={'Auth UI'}
        description={'A pre-built React component for authenticating users.'}
        style={{ height: '100%' }}
      />
    </div>    
    {/* Next.js */}
    <div class="col col--4">
      <ButtonCard
        class="card"
        to={useBaseUrl('/guides/auth/auth-helpers/nextjs')}
        title={'Next.js'}
        description={'Helpers for authenticating users in Next.js applications.'}
        style={{ height: '100%' }}
      />
    </div>
    {/* SvelteKit */}
    <div class="col col--4">
      <ButtonCard
        class="card"
        to={useBaseUrl('/guides/auth/auth-helpers/sveltekit')}
        title={'SvelteKit'}
        description={'Helpers for authenticating users in SvelteKit applications.'}
        style={{ height: '100%' }}
      />
    </div>
  </div>
</div>
```

File: apps/reference/docs/guides/auth/auth-helpers/nextjs.mdx [hunk 2]
```diff
@@ -0,0 +1,315 @@
+---
+id: nextjs
+title: Next.js Auth Helpers
+description: Authentication helpers for Next.js API routes, middleware, and SSR.
+---
+
+# Supabase Auth with Next.js
```
Comment (dannykng) [inline apps/reference/docs/guides/auth/auth-helpers/nextjs.mdx:7]: Can move the h1 heading to the front matter.
```suggestion
---
id: nextjs
title: Supabase Auth with Next.js
description: Authentication helpers for Next.js API routes, middleware, and SSR.
sidebar_label: "Next.js"
---
```

File: apps/reference/docs/guides/auth/auth-helpers/sveltekit.mdx [hunk 3]
```diff
@@ -0,0 +1,303 @@
+---
+id: sveltekit
+title: SvelteKit Auth Helpers
+description: Convenience helpers for implementing user authentication in SvelteKit.
+---
+
+# Supabase Auth with SvelteKit
```
Comment (dannykng) [inline apps/reference/docs/guides/auth/auth-helpers/sveltekit.mdx:7]: Can move the h1 heading to the front matter.
```suggestion
---
id: sveltekit
title: Supabase Auth with SvelteKit
description: Convenience helpers for implementing user authentication in SvelteKit.
sidebar_label: SvelteKit
---
```

**Comment by MildTomato** [comment]
Screenshot 2022-09-20 at 10 23 02 AM

ignore these

**Review by dannykng — APPROVED** [review]
@thorwebdev should we remove this content from the reference docs or have them in both places? Seems like we should just get rid of the Auth Helpers content in the reference docs until we have the generated content to put there.

Approved so I'm not blocking, but I disabled auto-merge to get your thoughts on this first 😅.

**Comment by thorwebdev** [comment]
I was thinking to keep it in both places for now as there's already Google listing for the reference docs wouldn't want these to go into the void. In a next step we'll replace the reference docs with the typedocs 👍

**Pull request merged**